tools/mason/mason.chpl

```diff
@@ -96,27 +96,40 @@ proc main(args: [] string) {
 
 
 proc masonClean() {
-  runCommand('rm -rf target');
+  try! {
+    const cwd = getEnv("PWD");
+    const projectHome = getTopLvlDirPath(cwd);
```

Comment:
Naming-wise, I find something like `getProjectHome()` easier to remember, as demonstrated by the choice of variable name here :)